app/assets/stylesheets/govuk_publishing_components/components/govspeak/_attachment.scss

```
@@ -56,6 +56,23 @@
 
         box-shadow: 0 2px 2px rgba(govuk-colour("black"), .4);
       }
+
+      svg {
+        display: block;
+        width: $thumbnail-width;
+        height: 140px;
+        border: govuk-colour("mid-grey", $legacy: "grey-3");
+        outline: $govuk-border-width solid transparentize(govuk-colour("black"), .9);
+        background: govuk-colour("white");
+        box-shadow: 0 2px 2px rgba(govuk-colour("black"), .4);
+        fill: govuk-colour("mid-grey", $legacy: "grey-3");
+        stroke: govuk-colour("mid-grey", $legacy: "grey-3");
+
+        @include govuk-if-ie8 {
+          max-width: none;
+          border: $govuk-border-width solid govuk-colour("mid-grey", $legacy: "grey-3");
+        }
+      }
```

Given that the change is basically a duplicate of the above, I would suggest doing the following:

Change [line 40](https://github.com/alphagov/govuk_publishing_components/pull/2788/files#diff-5315bc6594eb2695f5ff5d7a3585e58b342f1bf830fc16bb995f6e1dbb6eeb11R40) from `img` to `img, svg`.

and have the following changes for SVG specific:

```suggestion

      svg {
        fill: govuk-colour("mid-grey", $legacy: "grey-3");
        stroke: govuk-colour("mid-grey", $legacy: "grey-3");
      }
```